Resources/doc/reference/architecture.rst

```diff
@@ -325,7 +325,7 @@ To do this, you first need to call the ``addChild`` method in your PostAdmin ser
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
-        <service id="sonata.news.admin.post" class="Sonata\NewsBundle\Admin\PostAdmin">
+        <service id="sonata.news.admin.post" class="AppBundle\Admin\PostAdmin">
```

Comment:
not sure this should be changed, you can use this class, but if you want to change something, in your case the menu stuff, then you need a custom admin in userland ofc